electron/scripts/lib.js

```diff
@@ -3,8 +3,8 @@ var BASE_URL = "http://127.0.0.1:5000/";
 // Get the query parameters of the current page (not visable in electron)
 function getParameterByName(name, url) {
 	var regex = new RegExp("[?&]" + name + "=([^&#]*)(&|$|#)");
```

Comment:
Probably but I found stack overflow answer to copy first. I should maybe look for a library though, maybe later.